<ui version="4.0" >
 <class>LogConfig</class>
 <widget class="QWidget" name="LogConfig" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>347</width>
    <height>191</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Messages</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QTreeWidget" name="qtwMessages" >
     <property name="alternatingRowColors" >
      <bool>true</bool>
     </property>
     <property name="selectionMode" >
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="rootIsDecorated" >
      <bool>false</bool>
     </property>
     <property name="uniformRowHeights" >
      <bool>true</bool>
     </property>
     <property name="itemsExpandable" >
      <bool>false</bool>
     </property>
     <column>
      <property name="text" >
       <string>Message</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Console</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Notification</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Text-To-Speech</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Soundfile</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Path</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="qgbTTS" >
     <property name="title" >
      <string>Text To Speech</string>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <widget class="QLabel" name="qlVolume" >
        <property name="text" >
         <string>Volume</string>
        </property>
        <property name="buddy" >
         <cstring>qsVolume</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2" >
       <widget class="QSlider" name="qsVolume" >
        <property name="toolTip" >
         <string>Volume of Text-To-Speech Engine</string>
        </property>
        <property name="whatsThis" >
         <string>&lt;b>This is the volume used for the speech synthesis.&lt;/b></string>
        </property>
        <property name="maximum" >
         <number>100</number>
        </property>
        <property name="singleStep" >
         <number>5</number>
        </property>
        <property name="pageStep" >
         <number>20</number>
        </property>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition" >
         <enum>QSlider::TicksBelow</enum>
        </property>
        <property name="tickInterval" >
         <number>5</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="qlThreshold" >
        <property name="text" >
         <string>Length threshold</string>
        </property>
        <property name="buddy" >
         <cstring>qsbThreshold</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QSpinBox" name="qsbThreshold" >
        <property name="toolTip" >
         <string>Message length threshold for Text-To-Speech Engine</string>
        </property>
        <property name="whatsThis" >
         <string>&lt;b>This is the length threshold used for the Text-To-Speech Engine.&lt;/b>&lt;br />Messages longer than this limit will not be read aloud in their full length.</string>
        </property>
        <property name="buttonSymbols" >
         <enum>QAbstractSpinBox::PlusMinus</enum>
        </property>
        <property name="suffix" >
         <string> Characters</string>
        </property>
        <property name="maximum" >
         <number>5000</number>
        </property>
        <property name="singleStep" >
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="1" column="2" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>8</width>
          <height>16</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="mumble.qrc" />
 </resources>
 <connections/>
</ui>
